R/01-ckan.R

```diff
@@ -1,196 +1,242 @@
-#' Get CKAN Resource Choices
-#'
-#' Select, filter and sort choices to be available in the ckanResource input
-#'
-#' @param ckanResources (list) output of filterCKANFileList(getCKANFiles()) for a specific record
-#' @param types (character) user selected types to show
-#' @param sort (logical) if TRUE sort choices alphabetically
-getCKANResourcesChoices <-
-  function(ckanResources, types, sort = TRUE) {
-    if (is.null(ckanResources) || length(types) == 0) {
-      return(list(
-        choices = c("No resource available ..." = ""),
-        selected = c("No resource available ..." = "")
-      ))
-    }
-    # choices values
-    resources <- names(ckanResources)
-    # choices names to be displayed
-    labels <-
-      unlist(lapply(ckanResources, function(x) {
-        paste(x$name, " (", x$format, ")")
-      }))
-    
-    # available types
-    ckanTypes <- unlist(lapply(ckanResources, function(x) {
-      tolower(x$format)
-    }), use.names = FALSE)
-    # user filter for type
-    typesFilter <- ckanTypes %in% types
-    
-    if (all(!typesFilter)) {
-      return(list(
-        choices = c("Selected type(s) not available ..." = ""),
-        selected = c("Selected type(s) not available ..." = "")
-      ))
-    }
-    
-    # set choices
-    choices <- setNames(resources[typesFilter], labels[typesFilter])
+#' Get Repositories
+#' 
+#' Formerly getCKANResourcesChoices()
+#' 
+#' @param fileType (character) list of relevant file types, e.g. c("xls", "xlsx", "csv", "odt")
+#' @param repository (character) name of Pandora repository
+#' @param network (character) name of Pandora network
+#' @param pattern (character) string for meta information search
+#' @param order (logical) if TRUE order dataframe alphabetically by name
+#' 
+#' @return (data.frame) containing available resources within a repository
+#' @export
+getResources <- function(fileType = character(),
```

Comment:
is the Name a copy-paste mistake?

also i think is good to add a description (borrowed the text from your news.md)

```suggestion
#' Get Resources
#' 
#' Get all available resources within a repository or filtered
#' by file type or those within a specific network or within a specific repository
#' optional filtering of meta information for a given string
#'
#' Formerly getCKANResourcesChoices()
#' 
#' @param fileType (character) list of relevant file types, e.g. c("xls", "xlsx", "csv", "odt")
#' @param repository (character) name of Pandora repository
#' @param network (character) name of Pandora network
#' @param pattern (character) string for meta information search
#' @param order (logical) if TRUE order dataframe alphabetically by name
#' 
#' @return (data.frame) containing available resources within a repository
#' @export
getResources <- function(fileType = character(),
```